docs/developer-access-policy.md

```diff
@@ -0,0 +1,100 @@
+
+### What
+
+The OpenSAFELY team maintains a small set of test studies in the [OpenSAFELY Testing](https://jobs.opensafely.org/datalab/opensafely-testing/) project.
+These studies have pipelines that use the standard OpenSAFELY tooling,
+but which have been specially designed to test and assess the platform’s functionality rather than to answer research questions.
+
+Because they are standard OpenSAFELY pipelines, they benefit from all the same security controls as OpenSAFELY research pipelines,
+including publication of all code that is run, logging of all operations and manual checking of outputs for disclosivity before they are released from the backend.
+Access to the OpenSAFELY platform will also be required to verify the pipeline outputs are as expected.
+
+### Who
+
+The test studies can only be run by OpenSAFELY Core Developers.
+
+### When
+
+These test pipelines are run periodically as is necessary to check that the system is working properly.
+They are also run when updated software has been deployed;
+however this deployment only happens after new versions of the software have gone through careful testing with simulated data to ensure that they are functioning correctly and that the security characteristics of the platform are maintained.
+
+## _Ad hoc_ access to backend systems
+
+Note: this section applies to OpenSAFELY-TPP and OpenSAFELY-EMIS only.
+
+### Why
+
+In order to ensure the correct operation of the platform,
+it is sometimes necessary to directly access OpenSAFELY backend systems so that software can be reconfigured or updated and problems which only manifest themselves in the deployed system can be investigated.
+
+### What
+
+Access to backend systems is via secure, authenticated, encrypted channels. All access to the system and database operations are logged.
+This process gives access
+to the OpenSAFELY software running in the backend and logs that that software produces,
+to pseudonymized patient data,
+to unvetted intermediate data
+and to unchecked, unreleased study outputs.
+
+There is no facility for extracting data from the system via this process, it can only be studied within the system itself.
+
+### Who
+
+Direct access to the OpenSAFELY backend systems is only available to OpenSAFELY Core Developers.
+
+### When
+
+This process is routinely used for updating the platform with new software versions.
+It is exceptionally used for investigating outages or other problems, and only when the information necessary for the investigation can’t be accessed from outside the system.
+
+## Data curation activities
+
+Note: this section applies to OpenSAFELY-TPP and OpenSafely-EMIS only.
+
+### Why
+
+Characterizing new and updated datasets, or individual fields which were present but not previously in use, checking data quality and sense-checking.
```

Comment:
nit: This is slightly tricky to read for me: is "sense-checking" part of "checking data quality"? "sense-checking" might also be a little ambiguous.

Reply:
I've radically simplified this since most of it is duplicated in the "when" section below.